gallery/transforms/plot_transforms_illustrations.py

```diff
@@ -4,56 +4,34 @@
 ==========================
 
 .. note::
-    Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms.ipynb>`_
-    or :ref:`go to the end <sphx_glr_download_auto_examples_others_plot_transforms.py>` to download the full example code.
+    Try on `collab <https://colab.research.google.com/github/pytorch/vision/blob/gh-pages/main/_generated_ipynb_notebooks/plot_transforms_illustrations.ipynb>`_
+    or :ref:`go to the end <sphx_glr_download_auto_examples_transforms_plot_transforms_illustrations.py>` to download the full example code.
 
 This example illustrates the various transforms available in :ref:`the
 torchvision.transforms module <transforms>`.
 """
 
 # sphinx_gallery_thumbnail_path = "../../gallery/assets/transforms_thumbnail.png"
 
+# %%
 from PIL import Image
 from pathlib import Path
 import matplotlib.pyplot as plt
-import numpy as np
 
 import torch
-import torchvision.transforms as T
-
+from torchvision.transforms import v2
 
 plt.rcParams["savefig.bbox"] = 'tight'
+
+# If you're trying to run that on collab, you can download the assets and the
+# helpers from https://github.com/pytorch/vision/tree/main/gallery/
+from helpers import plot
 orig_img = Image.open(Path('../assets') / 'astronaut.jpg')
+
 # if you change the seed, make sure that the randomly-applied transforms
 # properly show that the image can be both transformed and *not* transformed!
 torch.manual_seed(0)
```

Comment:
Since this is now the last statement in this block, the return value is displayed

![image](https://github.com/pytorch/vision/assets/6849766/794fe40f-5e4c-4096-97e5-ead75b614382)


```suggestion
torch.manual_seed(0);
```